pywr/nodes.py

```diff
@@ -1074,4 +1074,96 @@ def after(self, timestep):
         self.commit_all(self.input.flow)
 
 
+class LossLink(Node):
+    """A node that has a proportional loss.
+
+    A fixed proportional loss of all flow through this node is sent to an internal output node. Max and min flows
+    applied to this node are enforced on the net output after losses. The node itself records the net output
+    in its flow attribute (which would be used by any attached recorders).
+
+    Parameters
+    ----------
+    model : `pywr.model.Model`
+    name : string
+        Name of the node.
+    loss_factor : float
```

Comment:
This probably should have a way of specifying the loss_factor based on gross inflow or net outflow, and constraints on gross inflow and/or net outflow.

Reply:
any plans to still do this?

Reply:
I'm not sure it's too tricky? What do you think?

Reply:
probably worth doing if its not too tricky. Does it just involve changing which component node commits flow to the parent node?

Reply:
I think you have to change which nodes are in the internal `AggregatedNode`. Or perhaps just the calculation of the factor.

Reply:
Let's leave it be for now.